frontend/packages/topology/src/components/side-bar/providers/SideBarTabLoader.tsx

```diff
@@ -15,7 +15,19 @@ import { getResource } from '@console/topology/src/utils';
 import { DefaultResourceSideBar } from '../DefaultResourceSideBar';
 import TopologyEdgeResourcesPanel from '../TopologyEdgeResourcesPanel';
 
-const TabSection: React.FC = ({ children }) => <>{children}</>;
+type TabSectionProps = {
+  element?: GraphElement;
+  sectionCallback?: any;
+  renderNull?: any;
+};
+const TabSection: React.FC<TabSectionProps> = observer(
+  ({ element, sectionCallback, renderNull }) => {
```

Comment:
When wrapping a component with `observer` we don't see component names anymore in the react dev tools. To avoid this and still the an component name you can use a inner function instead of an arrow function here:

```suggestion
const TabSection: React.FC<TabSectionProps> = observer(
  function TabSection({ element, sectionCallback, renderNull }) {
```